policyengine_us/parameters/gov/contrib/congress/golden/fisc_act/family_income_supplement/income_threshold.yaml

```
@@ -0,0 +1,10 @@
+description: Representative Jared Golden proposed a family income supplement which can not exceed this percentage of total adjusted gross income.
+metadata:
+  unit: /1
+  label: FISC Act family income supplement income threshold
+  period: year
```

right? or would we say year because it's based on annual agi?
```suggestion
  period: month
```

---

My thinking was to use the yearly period to the annual AGI amount - but open for discussion

---

Actually, will change to monthly since I am transitioning the entire Family Income Supplement Credit to a monthly variable to accommodate the pregnancy limitations